### DISCUSSION TITLE
Where metals user configuration file should be located

### DISCUSSION THREAD

**Discussion by kostafey** [scalameta/metals#6645] [Q&A] [answered]
How can I apply user configuration described in https://scalameta.org/metals/docs/editors/user-configuration
Where the actual configuration file should be located?

**Comment by tgodzik** [comment, marked as answer]
This depends on the editor, for example VS Code keeps the configuration in json files either in `.vscode` inside your workspace or under home directory. However, this differs from editor to editor

**Reply by kostafey** [reply]
Thank you. A description of the variables used for configuration, e.g. for Emacs (in my case) can be found at `lsp-metals.el`.
